sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/something.py

```diff
@@ -0,0 +1,14 @@
+b = bytearray(b'\x00S\x12\xd0\x00\x00\x01\n\x00\x00\x00\x0e\xa1$97176fc1-a225-4a22-bdb2-0db19a4aa569CAP\x00P\x00\x00S(\xc0=\x0b\xa10sender-link-97176fc1-a225-4a22-bdb2-0db19a4aa569@@@@@@@@@@\x00S)\xc0B\x07\xa19amqps://pytrack2eh.servicebus.windows.net/eh_for_livetest@@@@@@@@@\x80\x00\x00\x00\x00\x00\x01\x00\x00@@\xc1A\x02\xa3\x19com.microsoft:tracking-id\xa1#a19a961b19c64cf6adba81c5c8c0ffcd_G6')
```

Comment:
is this file for testing?

Reply:
nice catch, it's just a script for local test.
removed